### PR TITLE
Add new classification supplementary evidence with ocr

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-668

### Change description ###
- Update metafile schema to add classification `supplementary_evidence_with_ocr`
-  Functional test will be added in the next PR

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/bulk-scan-processor/bulk-scan-processor/pullRequest?prid=4539691&bid=15280048)

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java [hunk 1]
```diff
@@ -357,6 +357,37 @@ public void should_record_validation_failure_when_zip_duplicate_payment_dcns() t
         errorWasSent(SAMPLE_ZIP_FILE_NAME, ErrorCode.ERR_METAFILE_INVALID);
     }
 
+    @Test
+    public void should_record_validation_failure_when_zip_does_not_contain_required_document_type() throws Exception {
+        // given
+        uploadToBlobStorage(SAMPLE_ZIP_FILE_NAME, zipDir("zipcontents/missing_document")); // no "form" type document
```
Comment (apereverzin) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java]: Please, rename missing_document to a more descriptive

Comment (aliveni) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java]: renamed the test

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/bulk-scan-processor/bulk-scan-processor/pullRequest?prid=4539691&bid=15280048)

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/bulk-scan-processor/bulk-scan-processor/pullRequest?prid=4539691&bid=15280048)

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java [hunk 2]
```diff
@@ -357,6 +357,69 @@ public void should_record_validation_failure_when_zip_duplicate_payment_dcns() t
...
+
+        // then
+        envelopeWasNotCreated();
+        eventsWereCreated(ZIPFILE_PROCESSING_STARTED, FILE_VALIDATION_FAILURE);
+        fileWasDeleted(SAMPLE_ZIP_FILE_NAME);
+        errorWasSent(SAMPLE_ZIP_FILE_NAME, ErrorCode.ERR_METAFILE_INVALID);
+    }
+
+    @Test
+    public void should_reject_file_when_more_than_one_document_contains_ocr_data() throws Exception {
```
Comment (aliveni) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java]: This test is failing as the OcrValidator is mocked in the ProcessorTestSuite. I'll remove this test and the test data

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/bulk-scan-processor/bulk-scan-processor/pullRequest?prid=4539691&bid=15280048)

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/bulk-scan-processor/bulk-scan-processor/pullRequest?prid=4539691&bid=15280048)

**Pull request merged**